CHANGELOG.rst

```diff
@@ -1,6 +1,38 @@
 Changelog
 =========
 
+0.8.1 (2018-12-20)
+------------------
+
+MLflow 0.8.1 introduces several improvements:
+
+- Better UI performance, especially when navigating within experiments containing many runs, parameters, and metrics
```

Comment:
Let's say `(hundreds to thousands)` or something to indicate what "many" means. And maybe rather than "Better" "Improved UI responsiveness and load time" or something.

Reply:
Improved UI responsiveness and load time, especially when displaying experiments containing substantial numbers of runs, parameters, and metrics.

Reply:
Combined your suggestions! Let me know if it looks better!